site/content/blog/2019-12-20-onboarding-with-mattermost.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Onboarding with Mattermost"
+slug: onboarding-with-mattermost
+date: 2019-12-20T00:00:00-04:00
+categories:
+    - "announcements"
+author: CJ Wild
+github: imisshtml
+community: claude.wild
+---
+
+I recently joined Mattermost as a Remote Software Engineer and while I’ve worked in remote environments before, I was really surprised at how amazing this company is to work with. During my onboarding I discovered they have really developed a culture built around being Remote; one that allowed for me to feel as part of a larger team and always engaged, rather than separated from my peers and working alone. 
+
+### [Sponsored Simulation](https://docs.mattermost.com/process/developer.html#audition)
+The onboarding process at Mattermost was as unique as the company itself. As the company is fully remote, they took a great deal of care in making sure that not only was I a good fit, but that I would enjoy the environment and process that they instill. The first step was through a sponsored simulation, where I was given the opportunity to go beyond the normal open-source contributions and work directly with a mentor. 
+
+During the process, my mentor would help guide me in architecting the overall solution and in answering any questions I had. The community instance for Mattermost was also available, where I could post my questions to the core team and the open-source community. While I did not participate in any Scrum-related activities with the teams, I did follow their build and pull-request flow. The work overall was very fast-paced and responses to my questions and reviews of my PRs were surprisingly quick. Considering everyone is remote I was expecting the pace to be slower but was pleasantly surprised that I was helped and guided more quickly than I would have expected in a normal office environment.
+
+### [Detailed Onboarding Checklist and Follow-Up](https://handbook.mattermost.com/contributors/onboarding/staff-on-boarding-guide)
+One of the first things that occurred was that my mentor and a member of Human Resources created an extensive checklist for me to follow. This took me through all the starting documentation I would need to know and introduced me to videos and our development processes. This took me through my first two-weeks and made it easy to keep on track as I learned more about my duties and the company as a whole. 
+
+### [Extensive Resources](https://docs.mattermost.com/)
+There is a vast wealth of documentation available through the [Mattermost Documentation](https://docs.mattermost.com/), [Developer Documentation](https://developers.mattermost.com/), and their [Handbook](https://handbook.mattermost.com/). Everything from setting up your development environment and the Tick-Tock branching process to the setup and customization of the servers, apps and plugins. Many of these are part of the onboarding checklist, but it will take weeks to ingest all the knowledge!
```

Comment:
```suggestion
There is a wealth of documentation available through the [Mattermost Documentation](https://docs.mattermost.com/), [Developer Documentation](https://developers.mattermost.com/), and their [Handbook](https://handbook.mattermost.com/). Everything from setting up your development environment and the Tick-Tock branching process to the setup and customization of the servers, apps, and plugins. Many of these are part of the onboarding checklist, but it will take weeks to ingest all the knowledge!
```